src/troute-network/tests/test_HYFeaturesNetwork.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+from pathlib import Path
+import pandas as pd
+from troute.HYFeaturesNetwork import HYFeaturesNetwork
+from troute.routing.compute import compute_nhd_routing_v02
+#set the workdir relative to this test config
+#and use that to look for test data
+_workdir=Path(__file__).parent
+
+"""
+Fixtures for setting up various components for testing
+"""
+
+def supernetwork_parameters(type):
+    if( type == 'gpkg'):
+       geo_file =  _workdir.joinpath("data/hy_network.gpkg")
+       edge_list = None
+    elif( type == 'json'):
+        geo_file =  _workdir.joinpath("data/flowpath_attributes.json")
+        edge_list = _workdir.joinpath("data/flowpath_edge_list.json")
+    params = {
+        "title_string":"HY_Features Test",
+        "geo_file_path":geo_file,
+        "flowpath_edge_list":edge_list,
+        "columns": {
+            #link????
+            "key": "id",
+            "downstream": "toid",
+            "dx": "lengthkm",
+            "n": "n",  # TODO: rename to `manningn`
+            "ncc": "nCC",  # TODO: rename to `mannningncc`
+            "s0": "slope_percent",  # TODO: rename to `bedslope`
+            "bw": "BtmWdth",  # TODO: rename to `bottomwidth`
+            #waterbody: "NHDWaterbodyComID",
+            "tw": "TopWdth",  # TODO: rename to `topwidth`
+            "twcc": "TopWdthCC",  # TODO: rename to `topwidthcc`
+            #alt: "alt",
+            "musk": "MusK",
+            "musx": "MusX",
+            "cs": "ChSlp"  # TODO: rename to `sideslope`
+        },
+        "waterbody_null_code": -9999,
+        "terminal_code": 0,
+        "waterbody_null_code": -9999,
+        "driver_string": "NetCDF",
+        "layer_string": 0
+    }
+    return params
+
+@pytest.fixture
+def waterbody_parameters():
+    return {}
+
+@pytest.fixture
+def forcing_parameters():
+    return {
+        "nexus_input_folder": _workdir.joinpath("data"),
+        "nexus_file_pattern_filter": "nex-*"
+    }
+
+@pytest.fixture
+def restart_parameters():
+    return {}
+
+@pytest.fixture
+def network(request, waterbody_parameters, restart_parameters, forcing_parameters):
+    forcing = _workdir.joinpath("data/forcing/")
```

Comment:
@hellkite500 ,   is this neederd??

Reply:
Hmm...looks like it isn't used, so probably not needed haha.